src/shared/udev-util.h

```diff
@@ -46,3 +51,19 @@ int udev_resolve_subsys_kernel(const char *string, char *result, size_t maxsize,
 
 int udev_queue_is_empty(void);
 int udev_queue_init(void);
+
+#if HAVE_SYS_SDT_H
+
+#define DEVICE_TRACE_POINT(name, dev, ...)                                     \
+        do {                                                                   \
+                PROTECT_ERRNO;                                                 \
+                const char *_s = NULL;                                        \
```

Comment:
While you are at it: something went wrong with the placement of the line continuation `\` here. Maybe a tabs vs spaces problem?